### PR TITLE
Fix reference to dropped DNS.proposal_valid method

### PR DESCRIPTION
**Pull request by imobachgs**
Bring #74 to `master`.

### CODE REVIEW

File: src/clients/firstboot_hostname.rb [hunk 1]
```diff
@@ -94,9 +107,8 @@ def HostnameDialog
         Frame(
           _("Hostname and Domain Name"),
           VBox(
-            HBox("HOSTNAME", HSpacing(1), "DOMAIN"),
-            Left("DHCP_HOSTNAME"),
-            Left("WRITE_HOSTNAME")
+            Left("HOSTNAME"),
+            Left("DHCP_HOSTNAME")
```
Comment (dgdavid) [inline src/clients/firstboot_hostname.rb]: Just curiosity, why is this change needed?

Comment (imobachgs) [reply src/clients/firstboot_hostname.rb]: Because the hostname settings have been simplified. Now there are only two fields: _hostname_ and _set hostname via DHCP_. The _domain_ and _write hostname_ settings have been dropped.
